src/potions.ts

```diff
@@ -544,6 +614,30 @@ export function potionSetup(embezzlersOnly: boolean): void {
   }
 
   variableMeatPotionsSetup(0, embezzlers);
+
+  // Use paw wishes
+  while (get("_pawWishes", 0) < 5) {
+    // Sort the paw potions by the profits of a single wish, then use the best one
+    if (
+      !pawPotions
+        .filter((potion) => numericModifier(potion.effect(), "Meat Drop") >= 100)
+        .sort((a, b) => {
+          const aValue = a
+            .value(embezzlers)
+            .sort((l, r) => (r.quantity > 0 ? r.value : 0) - (l.quantity > 0 ? l.value : 0));
+          const bValue = b
+            .value(embezzlers)
+            .sort((l, r) => (r.quantity > 0 ? r.value : 0) - (l.quantity > 0 ? l.value : 0));
```

Comment:
this is hard to follow and probably should just use a `maxBy` given that you're ultimately checking their highest value